advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,591 @@
+---
+title: 'Multiple Repositories'
+description: "Multiple Repositories Support Feature"
+---
+
+The multiple repositories support feature was added in the [v2.33](https://github.com/pgbackrest/pgbackrest/releases/tag/release%2F2.33) release. You can use, e.g., this feature in a scenario where a local repository would have lesser retention (for fast restore) and a remote repository would have longer retention (for redundancy and access across the enterprise).
+
+A new [`--repo`](https://pgbackrest.org/configuration.html#section-repository/option-repo) option is added to set the repository. All `repo*-` options are indexed to enable configuring multiple repositories.
+
+In this section, we will set up a **demo** cluster (using **EDB Postgres Advanced Server** version 13 on CentOS 7) to demonstrate how this new feature impacts each pgBackRest command.
```

Comment:
Not sure the **emphasis** on demo is needed.

Reply:
I believe it's best to keep it since we'll use that keyword to name the stanza in the configuration file and commands...